README.md

```diff
@@ -1,24 +1,54 @@
 # Beyond-07-team-4
 
-**Reqirements:**
+**Prerequisite:**
 
-Adding a Vagrantfile for spinning up a development VM.
+VirtualBox: VirtualBox is a hypervisor, which lets us create and run virtual machines.
+The page below include links for downloading VirtualBox for various OS:
+https://www.virtualbox.org/wiki/Downloads. 
+During the installation process, Windows might pop-up some warning windows about installing 
+device driver software, please confirm for it that the software should be installed.
 
-Vagrant is a tool for using virtual machines to share development environments.
+Vagrant: Vagrant is a tool for using virtual machines to share development environments.
 Learn more about it at:
-https://www.vagrantup.com/
+https://www.vagrantup.com/. 
+Please make sure you have Vagrant installed on your computer, if not then install the latest version of the Vagrant for your current operating system:
+https://www.vagrantup.com/downloads. 
 
-Please make sure you have Vagrant installed on your computer, if not then you can download from here:
-https://www.vagrantup.com/downloads
+In order for it all to work, please enable virtualization hardware on your computer. An explanation on how to do it can be found here:
+https://www.virtualmetric.com/blog/how-to-enable-hardware-virtualization. 
 
-Also, make sure that you have VirtualBox installed. VirtualBox is a hypervisor, which lets us create and run virtual machines.
-The page below include links for downloading VirtualBox for various OS:
-https://www.virtualbox.org/wiki/Downloads
+**How to run a project in VagrantBox:**
 
-In order for it all to work, please enable virtualization hardware on your computer. An explanation on how to do it can be found here:
-https://www.virtualmetric.com/blog/how-to-enable-hardware-virtualization
+Create a folder in which the project will be placed.
+Open the terminal and cd to the relevant directory:
+
+*On Windows the commands to do so would be:
```

Comment:
This project is started with git clone, see you have some content in main branch

you should use the following:
```
git clone https://github.com/redhat-beyond/Beyond-07-team-4.git
cd Beyond-07-team-4
```

Since vagrantfile already exist there